src/main/java/org/jboss/xavier/sanitizer/XavierPayloadSanitizer.java

```diff
@@ -0,0 +1,169 @@
+{
+    private static String inputFileName;
+    private static String outputFileName;
+    private static String issuesConditionsJSONFileName;
+
+    public static void main(final String[] args)
+    {
+        List<String> argsList = Arrays.asList(args);
+        argsList.forEach(arg -> {
+            if( arg.equals("--input"))
+            {
+                inputFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+            if( arg.equals("--output"))
+            {
+                outputFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+
+            if( arg.equals("--issues"))
+            {
+                issuesConditionsJSONFileName = argsList.get(argsList.indexOf(arg) + 1);
+            }
+        });
+
+        String jsonFileToBeSanitized = readInputFile(inputFileName, "Input file to sanitize");
+
+        String jsonIssuesConditions = readInputFile(issuesConditionsJSONFileName, "Issues Conditions file ");
+        List<String> vmsIssues = splitConditions(jsonIssuesConditions,"###vms");
+        List<String> hostsIssues = splitConditions(jsonIssuesConditions,"###hosts");
+        List<String> clustersIssues = splitConditions(jsonIssuesConditions,"###clusters");
+
+        String outputJsonString = sanitize(jsonFileToBeSanitized, vmsIssues, hostsIssues,clustersIssues);
+
+        writeStringToFile(outputFileName, outputJsonString);
+
+    }
+
+    private static String readInputFile(String inputFile, String description)
+    {
+        if (inputFile == null)
+        {
+            throw new RuntimeException(description + "path not specified");
+        }
+        try
+        {
+            return new String(Files.readAllBytes(Paths.get(inputFile)), StandardCharsets.UTF_8);
+        }
+        catch(IOException ioe)
+        {
+            throw new RuntimeException(description + "could not be read");
+        }
+    }
+
+
+
+    private static List<String> splitConditions(String inputConditionsString, String sectionHeader)
+    {
+        if (inputConditionsString == null)
+        {
+            throw new RuntimeException("No issues conditions found");
+        }
+        List<String> returnIssues = new ArrayList<>();
+        String[] issues = inputConditionsString.split("\\R");
+        boolean currentSectionWanted = false;
+        for (String issue:issues)
+        {
+            if(issue.equals(sectionHeader))
+            {
+                currentSectionWanted = true;
+                continue;
+            }else if (issue.startsWith("###"))
+            {
+                currentSectionWanted = false;
+                continue;
+            }
+            if(currentSectionWanted)
+            {
+                returnIssues.add(issue);
+            }
+        }
+
+        return returnIssues;
+
+    }
+
+    private static String sanitize(String fileToBeSanitized, List<String> vmsIssuesConditions, List<String> hostsIssuesConditions, List<String> clustersIssuesConditions)
```

Comment:
Before or After calling this method I suggest to print the list of VMs that will be marked as retired, just to let know the user that the Sanitizer found some problems. Something like: `Found some problems with VMs: 1,5,7. They will be marked as retired` (this message is just a suggestion, you can think of a better message)